### PR TITLE
YONK-1770 Upgrading to PY 3.5

### PR DESCRIPTION
**Pull request by ihtram**
run modernize 
run pyupgrade to remove py2 support
run isort

### CODE REVIEW

File: edx_solutions_projects/management/commands/revert_rename_projects_app.py [hunk 1]
```diff
@@ -2,9 +2,10 @@
 Management command to revert renaming of projects app
 """
 import logging
-from south.db import db
+
 from django.core.management.base import BaseCommand
 from django.db import transaction
+from south.db import db
```
Comment (shafqatfarhan) [inline edx_solutions_projects/management/commands/revert_rename_projects_app.py]: We should update this old syntax to `from django.db import connection` to execute query

File: setup.py [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
+    version='4.0.0',
     description='Projects management extension for edX platform',
     long_description=open('README.rst').read(),
     author='edX',
     url='https://github.com/edx-solutions/projects-edx-platform-extensions.git',
     packages=find_packages(),
     include_package_data=True,
     install_requires=[
-        "Django>=1.11,<1.12",
+        "Django==2.2.15",
```
Comment (shafqatfarhan) [inline setup.py]: `"Django>=2.2,<3.0",` is better

**Review by shafqatfarhan — APPROVED** [review]
Left 2 comments, other than that, LGTM

File: setup.py (hunk 2 above)
Comment (ihtram) [reply setup.py]: done

**Review by mzulqarnain1 — APPROVED** [review]

**Review by attiyaIshaque — APPROVED** [review]
looks good 👍

**Pull request merged**